docs/user_interface.rst

```diff
@@ -25,284 +24,412 @@ If you do processing or acquisition which produces new data, its associated disp
 
-You can drag items from the data panel to display panels in the workspace.
+Having individual utility panels for similar parameters means that the workspace can be fully customized for the user's purpose. There are several ways to customize the utility panel layout:
 
-You can delete display items (and associated data items if they become orphaned) by selecting them in the data panel and pressing the delete key.
+* Choose to show or hide any utility panel by using the [Window] menu.
 
-You can filter (by text) what items appear in the data panel by typing into the :guilabel:`Filter` field at the bottom of the data panel. The filter works on the title and the size/data type fields.
+* Quickly close a utility panel by clicking the x button in the corner of its title bar.
 
-.. _Output Panel:
+* Click and drag a utility panel by its title bar to rearrange the layout of the shown panels.
 
-Output
-------
-The Output panel (:menuselection:`Window --> Output`) shows text output while running the application.
+* Layer utility panels by dropping one utility panel onto another. If two or more panels are layered, selection buttons will appear at the bottom of the panel to switch between them.
 
-Additional debugging information may be available using debugger consoles if launched from the console.
+* Pop utility panels out into a separate window by dragging the panel away from the main window by the title bar.
 
-.. _Info Panel:
+* Resize a utility panel by clicking and dragging from its edges.
 
-Info
-----
-The Info panel (:menuselection:`Window --> Info`) shows the position of the cursor and the value at the pixel under the cursor if the cursor is over a display item showing a single data item in a display panel. It will also show the intensity level of the data represented at the cursor position in the histogram panel.
+.. image:: graphics/customize_utility_panels.png
+    :width: 396
+    :alt: Customizing Utility Panels
+    :align: center
 
-.. _Histogram Panel:
+The utility panels are organized alphabetically in the [Window] menu, and their functions are as follows:
 
-Histogram
----------
-The Histogram panel (:menuselection:`Window --> Histogram`) shows the histogram graph and other statistics of the data in the selected display panel if a display item with a single data item is shown. It also shows the mapping of data intensity values to colors in a color bar just below the graph.
+.. _Activity Panel:
 
-If there are either zero or more than one display panels selected or if the display item in the selected display panel displays more than one data item, the histogram will be blank.
+Activity
+========
+.. image:: graphics/activity_panel.png
```

Comment:
This is a formatting review -- so I'm only discussing formatting. But I'd like to have the section header, an introductory paragraph, then an explanation of the details of the panel. This goes for all panels in this document. Having the header followed immediately by the image is difficult to process visually...